resources/models/resource.py

```diff
@@ -391,13 +393,39 @@ def get_reservable_days_in_advance(self):
     def get_reservable_before(self):
         return create_reservable_before_datetime(self.get_reservable_days_in_advance())
 
+    def get_supported_reservation_extra_field_names(self):
+        if not self.reservation_metadata_set:
```

Comment:
Yes it does, but we cannot use `metadata_set.values_list()` if `metadata_set` is `None`